apiserver/facades/agent/uniter/networkinfocaas.go

```diff
@@ -115,7 +115,7 @@ func (n *NetworkInfoCAAS) ProcessAPIRequest(args params.NetworkInfoParams) (para
 			info.EgressSubnets = subnetsForAddresses(info.IngressAddresses)
 		}
 
-		result.Results[endpoint] = info
+		result.Results[endpoint] = n.resolveResultHostNames(info)
```

Comment:
for k8s we don't want to resolve ingress addresses

Reply:
Yes, I will introduce this logical branching subsequently.